src/applications/simple-forms/shared/config/submit-transformer.js

```diff
@@ -1,9 +1,25 @@
 import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
 
+const escapedCharacterReplacer = (_key, value) => {
+  if (typeof value === 'string') {
+    return value
+      .replaceAll('"', "'")
+      .replaceAll('/', '[forwardslash]')
```

Comment:
is using forward slash an issue? because this takes up quite a lot of characters. I would vote to keep "/" if it's not an issue, and even convert all backslash to "/" as well. if we could simplify like this, then we also could simplify the last regex.

Reply:
That's a great question, I posted about it in slack. I think I need @Thrillberg to throw some test data at that ERB result method to see what it does with forward slash. I read two conflicting pieces of information about forward slash where one said it did need to be escaped for JSON and the other said it didn't. From testing locally, I don't _think_ it needs to be escaped, but again I'd like to confirm with Eric to be 100% sure. But I totally agree with you Robert, if it doesn't need to be escaped, then we'll just leave it as it was typed and convert all back slashes to forward slashes.

Reply:
Here's the test data I entered:
<img width="252" alt="Screenshot 2023-08-01 at 10 08 30 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/10481391/d6bdabd7-f17e-43fb-8d70-dc46891b431e">

Here's the output right off the PDF:
<img width="265" alt="Screenshot 2023-08-01 at 10 08 14 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/10481391/f1b7202d-64bf-4ffd-b634-e8e6d4fb142e">

Looks good to me! 👍